### DISCUSSION TITLE
Raspeberry 4 + HASSOS, continui crash

### DISCUSSION THREAD

**Discussion by 50494554524F** [indomus/forum#722] [HUB: Home Assistant]
è da qualche giorno che continua a crashare, non ho fatto modifiche... si è solo auto aggiornato il supervisor.

ho cambiato microsd. messo nuove versioni e vecchie versioni. niente da fare.

qualcuno ha questo problema?! se vi funziona tutto bene mi dite che versioni state usando?! così vado a controllare le integrazioni / componenti una ad una...

ora ho rimesso l'ultima versione senza niente, vediamo un po' se dura.

grazie

**Comment by 50494554524F** [comment]
senza niente non si è bloccato, comincio a rimettere un pezzettino alla volta... speriamo che sia il solito aggiornamento incasinatore, che noia però

**Comment by 50494554524F** [comment]
niente da fare, continua a crashare appena rimetto qualcosa... hanno fatto il danno

**Reply by indomus** [reply]
L'ultima release di Core aveva problemi con la penultima di HASSOS, su Raspberry. Installa l'ultimissima versione di quest'ultimo (6.3), non dovresti avere problemi.

**Reply by 50494554524F** [reply]
avevo os 5.13 e core 2019.7 quando ha cominciato a dare problemi, non ho cambiato niente... poi ho provato un po' di tutto... l'unica cosa che non si può cambiare è il supervisor, mi sa che è qui il problema...

**Reply by 50494554524F** [reply]
6.3 è quella che stavo usando ieri, è scritto nel primo post...

**Reply by indomus** [reply]
pardon. e allora o è guasto RPI, o stai sbagliando qualcosa tu.

**Reply by 50494554524F** [reply]
non ho cambiato niente, tanto che lo avevo lasciato acceso senza più fare modifiche per testare in vista dell'inverno ed era durato un mese... ora sto pensando a cosa può essere cambiato oltre al supervisor... può essere hacs?! di integrazioni ho (ri)aggiunto solo ble_monitor, ma crasha anche con una vecchissima versione... ho spostato il raspberry, ma è più vicino al router... altro non c'è, nei log non c'è niente... che altro posso guardare?!

**Reply by snakuzzo** [reply]
@50494554524F se hai ancora la versione di Core "buggata"...

Rinomina il file `automations.yaml` via SMB o SSH
Dovrebbe ripartire in safe mode e da shell puoi fare l'update di HA

`➜  ~ ha core update`

A quel punto puoi fare il restore del backup (se ce l'hai) o rinominare nuovamente in maniera corretta il file `automations.yaml`

**Reply by 50494554524F** [reply]
@indomus : proverò, vediamo...

@snakuzzo : ho già l'ultima versione ora; non sto usando automations.yaml ma include_dir ...


edit: cut... risolto, piccolo bug nel frontend...

**Reply by snakuzzo** [reply]
guarda...dopo aver letto gli "sfoghi" di prima e dopo quest'ultimo commento, preferirei non andare oltre

**Comment by 50494554524F** [comment]
usando un dongle bluetooth sembra che non si blocchi più... ora devo vedere se si è rotto qualcosa o se dipende dagli ultimi aggiornamenti

**Comment by 50494554524F** [comment]
riprovando tutte le vecchie versioni ho risolto, spero, usando haos 5.11... core e supervisor possono essere aggiornate all'ultima versione... quindi deve essere un problema di driver che colpisce un po' a caso

**Reply by snakuzzo** [reply]
1. Non creare sempre nuovi thread ma rispondi sotto l'ultimo messaggio
2. Il "caso" non esiste...
3. Al posto di fare prove a caso, controlla le issue di HassOS e nel caso aprine una tu

**Reply by 50494554524F** [reply]
è problema casuale perché non capita a tutti... non sono prove a caso, ho (ri)trovato la versione funzionante e dedotto che è un problema di driver... oltre questo le mie conoscenze di linux non vanno, nei log non c'è niente

**Reply by snakuzzo** [reply]
Io continuo a dirti che in informatica NULLA è casuale e ti invito ad aprire una issue su github. Io lo dico per te...a me non è che interessi più di tanto. Se vuoi seguire il consiglio, bene...altrimenti fai come meglio credi e affidati al caso.

**Reply by 50494554524F** [reply]
e che dovrei scriverci quando mi chiederanno in quali situazioni crasha o del perché crasha?! non lo so... ce ne sono centinaia di utenti con questo tipo di problemi ma ancora non si trova una soluzione, c'è chi prova con le medesime configurazioni e non ha problemi... sennò avrei già risolto...

vabbè, adesso mi serve per le valvole... congelo tutto e se ne riparla a primavera, basta che funzioni

**Reply by snakuzzo** [reply]
contribuisci con il tuo caso...scrivi quello che ti chiederanno (eventualmente) di verificare. Sii parte attiva e non aspettare solo alla finestra sperando che qualcuno con la tua stessa situazione risolva. Tutto qua
